content/doc/book/using/best-practices.adoc

```diff
@@ -0,0 +1,177 @@
+
+.Sending Slack notifications
+video::EDVZli8GdUM[youtube,width=800,height=420]
+
+=== Use simple project names
+
+Jenkins uses project names for folders related to the project.
+Many tools cannot handle spaces, dollar signs, or similar characters in file paths.
+Limit yourself to alphanumeric characters [a-zA-Z0-9_-]+ in project names.
+Use the *Display Name* feature to make them look nice.
+The "Restrict project naming" setting in the system configuration can enforce naming restrictions for all projects.
+
+=== Fingerprint your dependencies
+
+When you have interdependent projects, it often becomes hard to keep track of which version of this is used by which version of that.
+Jenkins supports "file fingerprinting" to simplify this.
+
+More information is available on the link:/doc/book/using/fingerprints/[fingerprinting page].
+
+=== Don't use the Maven job type
+
+Jenkins has provided the plugin:maven-plugin[Maven integration plugin] for many years.
+Jobs created with the Maven integration plugin are created with the "Maven project" selection from the Jenkins "New item" menu.
+The Maven job type provides a higher level of integration with maven builds, but creates unnecessary complications due to that higher level of integration.
+
+Use organization folders, multibranch Pipeline, or Pipeline jobs instead of the Maven job type.
+
+The Jenkins project uses organization folders to build link:https://ci.jenkins.io/job/Core/[Jenkins core] and link:https://ci.jenkins.io/job/Plugins/[Jenkins plugins] on ci.jenkins.io.
+A Jenkins Pipeline builds Maven projects easily and provides much better control for Maven users.
+
+Refer to the link:https://plugins.jenkins.io/maven-plugin/#plugin-content-risks[Maven plugin documentation] for more details.
+
+== Manage your controller
+
+The Jenkins controller is a central resource that needs to be managed for best results.
+These practices can help your controller provide the best experience for users.
+
+=== Secure the controller
+
+Jenkins installations have security enabled by default.
+However, it is possible to disable security.
+Do **not** disable security.
+
+Details are in the link:/doc/book/security/securing-jenkins/[securing Jenkins] chapter of the User Handbook.
+
+=== Back up regularly
+
+Even the most reliable of systems will have failures.
+Prepare and regularly check the health of your backups.
+
+More details can be found in the link:/doc/book/system-administration/backing-up/[backup documentation].
+
+=== Avoid scheduling overload
+
+Schedule jobs to balance the number of jobs running concurrently.
+If you're using timer triggers or are periodically polling, use the `H` syntax in the cron expression to introduce scheduing jitter.
+The predefined tokens such as `@hourly` can also be used to distribute job starting times more evenly.
+
+=== Avoid resource collisions
+
+Multiple jobs running at the same time often cause collisions if they set up a service or need exclusive access.
```

Comment:
```suggestion
When multiple jobs run simultaneously, there is a possibility of collisions occurring, especially if they require exclusive access to certain resources or set-up services.
To prevent interference and ensure smooth execution, it is important to manage resource access effectively.
```